src/tests/functional/plugin/shared/include/multi/multi_remote_blob_multidevice_test.hpp

```diff
@@ -80,3 +76,252 @@ TEST_P(MultiDeviceMultipleGPU_Test, canCreateRemoteTensorThenInferWithAffinity)
         }
     }
 }
+
+TEST_P(MultiDeviceCreateContextMultipleGPU_Test, canInferOnUserContextWithSystemMemory) {
+    auto ie = ov::Core();
+    ov::CompiledModel exec_net_regular;
+    try {
+        exec_net_regular = ie.compile_model(function, device_names);
+    } catch (...) {
+        // device is unavailable (e.g. for the "second GPU" test) or other (e.g. env) issues not related to the test
```

Comment:
Maybe can add some mock tests for multi-gpu.

Reply:
Any updates on this PR? Thanks!

Reply:
convert to draft, this will need more clear use case definition